tools/CodeGenerator/KnownHeaders.cs

```diff
@@ -269,6 +269,21 @@ public static string GeneratedFile()
                 PrimaryHeader = responsePrimaryHeaders.Contains("Content-Length")
             }})
             .ToArray();
+
+            var responseTrailers = new[]
+            {
+                "ETag",
```

Comment:
Has anyone asked to send an ETag in the trailer? Do any clients support this? Adding "grpc-status" and "grpc-message" might be worthwhile.

Reply:
ETag is a commonly discussed example for trailers, though it's not clear where it's supported. I added it here primarily as a placeholder to make the code generator work correctly. I hesitate to add GRPC specific headers, it's not clear how final that spec is.